### DISCUSSION TITLE
vscode插件如何提供debug时的argv参数？

### DISCUSSION THREAD

**Discussion by nihao996** [xmake-io/xmake-vscode#126] [Q&A] [answered]
在哪里能配置呢？

**Comment by waruqi** [comment, marked as answer]
https://github.com/xmake-io/xmake-vscode/blob/ad72fc457f386803300ea37d9b0d02534232c0b0/package.json#L497-L503

**Reply by wanzzhehe** [reply]
请问如何对多个target特定设置参数呢？

**Reply by wanzzhehe** [reply]
啊，我看懂了

**Reply by wanzzhehe** [reply]
<img width="1281" alt="image" src="https://user-images.githubusercontent.com/26498686/232265255-a27d7b9f-1682-433f-b23f-4d48353b83a3.png">
<img width="1286" alt="image" src="https://user-images.githubusercontent.com/26498686/232265268-99a4f634-a93b-4352-9ba2-6edb6abc8e20.png">

请问为什么添加完参数之后无法进行调试？
